command/config/update.go

```diff
@@ -65,6 +63,15 @@ var CommandUpdate = &cli.Command{
 			Usage:   "update the log level in the config file",
 		},
 
+		// Git Sync Flags
+
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_GIT_SYNC", "CONFIG_GIT_SYNC", "GIT_SYNC"},
```

Comment:
Should the environment variables and usage instructions be updated to reference `noGit` instead of `git sync`?